Makefile

```diff
@@ -107,95 +107,106 @@ CPPFLAGS=-std=c++11 -fPIC $(LFS_CFLAGS) $(CFLAGS)
+  LDFLAGS:= $(TILEDB_DIR)/core/lib/$(TILEDB_BUILD)/libtiledb.a $(LDFLAGS)
 else
-    LDFLAGS:= -Wl,-Bstatic -L$(TILEDB_DIR)/core/lib/$(TILEDB_BUILD) -ltiledb -Wl,-Bdynamic $(LDFLAGS)
+  LDFLAGS:= -Wl,-Bstatic -L$(TILEDB_DIR)/core/lib/$(TILEDB_BUILD) -ltiledb -Wl,-Bdynamic $(LDFLAGS)
 endif
 
 # --- Htslib Source --- #
 HTSLIB_BUILD_NUM_THREADS ?= 1
 HTSLIB_EXTRA_CFLAGS=
 ifndef HTSDIR
-    HTSDIR=dependencies/htslib
+  HTSDIR=dependencies/htslib
 endif
 ifdef HTSDIR
-    CPPFLAGS+=-I$(HTSDIR) -DHTSDIR
-    ifeq ($(OS), Darwin)
-        LDFLAGS:=$(HTSDIR)/libhts.a $(LDFLAGS)
-        HTSLIB_EXTRA_CFLAGS=-mmacosx-version-min=10.9
-    else
-        LDFLAGS:=-Wl,-Bstatic -L$(HTSDIR) -lhts -Wl,-Bdynamic $(LDFLAGS)
-    endif
+  CPPFLAGS+=-I$(HTSDIR) -DHTSDIR
+  ifeq ($(OS), Darwin)
+    LDFLAGS:=$(HTSDIR)/libhts.a $(LDFLAGS)
+    HTSLIB_EXTRA_CFLAGS=-mmacosx-version-min=10.9
+  else
+    LDFLAGS:=-Wl,-Bstatic -L$(HTSDIR) -lhts -Wl,-Bdynamic $(LDFLAGS)
+  endif
 endif
 
 # --- RapidJSON Source (header only library) --- #
 ifndef RAPIDJSON_INCLUDE_DIR
-    RAPIDJSON_INCLUDE_DIR=dependencies/RapidJSON/include
+  RAPIDJSON_INCLUDE_DIR=dependencies/RapidJSON/include
 endif
 CPPFLAGS+=-I$(RAPIDJSON_INCLUDE_DIR)
 
 #libcsv - optional, but required if csvs need to be imported
 ifdef LIBCSV_DIR
-    CPPFLAGS+=-DUSE_LIBCSV -I$(LIBCSV_DIR)
-    LDFLAGS+=-L$(LIBCSV_DIR)/.libs -L$(LIBCSV_DIR)/lib -lcsv
+  CPPFLAGS+=-DUSE_LIBCSV -I$(LIBCSV_DIR)
+  LDFLAGS+=-L$(LIBCSV_DIR)/.libs -L$(LIBCSV_DIR)/lib -lcsv
 else
-    ifdef USE_LIBCSV
-	CPPFLAGS+=-DUSE_LIBCSV
-	LDFLAGS+=-lcsv
-    endif
+  ifdef USE_LIBCSV
+    CPPFLAGS+=-DUSE_LIBCSV
+    LDFLAGS+=-lcsv
+  endif
 endif
 
 # --- JNI flag - optional, but required if the JNI library is needed --- #
 ifdef JNI_FLAGS
-    CPPFLAGS+=$(JNI_FLAGS)
+  CPPFLAGS+=$(JNI_FLAGS)
 endif
 
 # --- BigMPI Flags (optional) --- #
 ifdef USE_BIGMPI
-    CPPFLAGS+=-I$(USE_BIGMPI)/src -DUSE_BIGMPI
-    LDFLAGS+=-L$(USE_BIGMPI)/src -lbigmpi
+  CPPFLAGS+=-I$(USE_BIGMPI)/src -DUSE_BIGMPI
+  LDFLAGS+=-L$(USE_BIGMPI)/src -lbigmpi
 endif
 
 ifdef DO_PROFILING
-    CPPFLAGS+=-DDO_PROFILING
+  CPPFLAGS+=-DDO_PROFILING
 endif
 
 ifdef DO_MEMORY_PROFILING
-    CPPFLAGS+=-DDO_MEMORY_PROFILING
+  CPPFLAGS+=-DDO_MEMORY_PROFILING
 endif
 
 # --- Google Performance Tools Library (optional) --- #
 ifdef USE_GPERFTOOLS
-    ifdef GPERFTOOLSDIR
-	CPPFLAGS+=-DUSE_GPERFTOOLS -I$(GPERFTOOLSDIR)/include
-	LDFLAGS += -Wl,-Bstatic -L$(GPERFTOOLSDIR)/lib -lprofiler -Wl,-Bdynamic  -lunwind
-    endif
+  ifdef GPERFTOOLSDIR
+    CPPFLAGS+=-DUSE_GPERFTOOLS -I$(GPERFTOOLSDIR)/include
+    LDFLAGS += -Wl,-Bstatic -L$(GPERFTOOLSDIR)/lib -lprofiler -Wl,-Bdynamic  -lunwind
+  endif
 endif
 
 ifdef VERBOSE
-    CPPFLAGS+= -DVERBOSE=$(VERBOSE)
+  CPPFLAGS+= -DVERBOSE=$(VERBOSE)
 endif
 
+# --- Additional load flags for protocol buffers and google test --- #
+GOOGLETEST_LIB=$(HOME)/workspace/googletest/googletest
```

Comment:
This is required for C++ unit tests written using Google test fixtures. Optional now, so we can remove it.